dipy/data/fetcher.py

```diff
@@ -431,6 +432,27 @@ def fetcher():
          " https://www.nature.com/articles/sdata201672"))
 
 
+fetch_qtdMRI_test_retest_2subjects = _make_fetcher(
+    "fetch_qtdMRI_test_retest_2subjects",
+    pjoin(dipy_home, 'qtdMRI_test_retest_2subjects'),
+    'https://zenodo.org/record/996889/files/',
```

Comment:
Strangely, every time I run the fetcher it downloads the data again, even though I already have the data. Could you please check that you have this implemented like the other fetchers/readers? I think there's something that should check if the files are already there, but I don't remember exactly how that works

Reply:
The data is released under Creative Commons Attribution, meaning it can be used for anything as long as they at least give credit and cite the data source. I don't see how that can conflict with any future application?

The fetcher takes some option "md5_list". I didn't quite understand what that meant so I didn't fill it. Could it be related to the fact that it keeps downloading? How should I remedy this?

Reply:
It's a list of the md5 hashes of the files. We use that to make sure that the file contents are not corrupted on download. You can get the md5 of a file by running this function on it: https://github.com/nipy/dipy/blob/master/dipy/data/fetcher.py#L90

Regarding the license - not a big deal. The main issue is ambiguity about how to handle situations in which the data is "hidden". For example in (the hypothetical) software that automatically derives only a single voxel worth of data, and does some transformation on it, before returning it to the user. What is the user's obligation in that case. But like I said -- if this rubs anyone the wrong way, it's fine by me. The license you have there is quite liberal as is. If it's all the same to you, you can change the license, and prevent these kinds of ambiguities from occurring in the future. Even if there is no legal obligation to cite you, any reasonable scientist would still cite where there data comes from, if only so that the provenance of the data is clear.

Reply:
I just fixed the md5 hashes so the function won't keep downloading the dataset.

As for the license, I emailed Zenodo if I can choose a CC0 or PDDL license since they are non-selectable from the available licenses. If they get back to me with a positive response I will change the license to one of these open domain ones.

Reply:
As requested, the data is now licensed under CC0.

Reply:
Fantastic! Thanks for doing that.